docs/recommendations/recommendations.md

```diff
@@ -144,11 +144,100 @@ getRecommendations({
   "statusCode": 200
 }
 ```
-> [recommendation](#recommendation) in `results`
+> [recommendation](#recommendation) in this case is the response above
 
 #### Result response
 | Name                 | Description                              | Type     |
 |----------------------|------------------------------------------|----------|
 | dataIdExternal       | The client’s id for the recommended item | `string` |
 | recommendationWeight | The recommender’s weight for the item    | `float`  |
 | additionalData       | Additional data supplied by the client   | `object` |
+
+## Carousel
+Our Carousel extends [react-slick](https://react-slick.neostack.com/docs/api).
+
+| Name                                                                                  | Description                                                                                                 | Type                  | Props            |
+|---------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------|-----------------------|------------------|
+| **recommendationQuery**                                                               | Query to call the [Breinify API](https://docs.breinify.com/#recommendations)                                | `object`              |                  |
+| **containerClassName**                                                                | Container className to overwrite the default className (`breinify-carousel`)                                | `string`              |                  |
+| **containerStyles**                                                                   | Styles to pass into the carousel container                                                                  | `React.CSSProperties` |                  |
+| **onError**                                                                           | A callback when `getRecs` receives an error                                                                 | `function`            | (`error`: `any`) |
+| **buttonLabel**                                                                       | Change the label name of the carousel item's button                                                         | `string`              |                  |
+| **onButtonClick**                                                                     | A callback when an item in the carousel is clicked. It provides a filtered data through `getComponentProps` | `function`            | (`props`: `any`) |
```

Comment:
I think we should define better what `filtered data` here is, and I'm not sure how `getComponentProps` fits in. To my knowledge, `onButtonClick` should call the function passed in with the `recommendation` that is displayed. Is that correct?

Reply:
`onButtonClick` passes back all the data that was filtered through `getComponentProps` (title, image, description).

Reply:
What do you think about `onButtonClick` passing back the entire recommendation entry? I feel like mixing `props` as the parameter is a little weird since `props` are usually associated with how the component will render. I can also see a scenario where the user might want something from the recommendation that they don't want to use as props

Reply:
Yeah that sounds good, I'm gonna add the changes in this PR

Reply:
Looks good! Approved 😄